docs/about/showcase.soy

```diff
@@ -184,6 +184,58 @@
 {/call}
 <!-- End Brainium Studios Apps -->
 
+<!-- Uber Apps -->
+{call .company}
+  {param name: 'Uber' /}
+  {param moreInfo: quoteKeysIfJs([
+      'Uber Engineering's iOS Monorepo:
+        'https://eng.uber.com/ios-monorepo/',
+    ]) /}
+  {param items}
+    {call .item}
+      {param name: 'Uber' /}
+        {param imgSrc: 'http://is3.mzstatic.com/image/thumb/Purple111/v4/32/36/9a/32369abe-e5a8-60b2-1d90-9322a7c87aa9/source/175x175bb.jpg' /}
```

Comment:
We serve our pages over SSL, and having a non-SSL image url will break that and show mixed content warnings.  This is why the Facebook stuff references images from the play store instead of some other random place.  Can you please find an SSL-served image?